packages/smarthome/smartbase0.py

```diff
@@ -45,3 +48,15 @@ def checksend(self, answer):
             log.warning("(" + str(self.device_nummer) +
                         ") checksend Fehlermeldung: %s "
                         % (str(e1)))
+
+    def callpro(self, argumentList: List[str]) -> None:
+        try:
+            my_env = os.environ.copy()
+            my_env["PYTHONPATH"] = "/var/www/html/openWB/packages"
+            proc = subprocess.Popen(argumentList, stderr=subprocess.PIPE,
+                                    universal_newlines=True, env=my_env)
+            _, errs = proc.communicate()
+            if errs:
+                log.error("%s" % (errs))
+        except Exception as e1:
+            log.warning("Subprocess Fehlermeldung: %s in %s " % (str(e1)), argumentList[1])
```

Comment:
```suggestion
        except Exception:
            log.exception("Subprocess Fehlermeldung: argumentList %s " % argumentList[1])
```
Logmeldungen aus allgemeinen Exceptions bitte immer mit log.exception loggen, damit sie immer geloggt werden.